modules/coverage-reports/pom.xml

```diff
@@ -3,7 +3,7 @@
     <parent>
         <artifactId>secrets-manager-sdk</artifactId>
         <groupId>com.ibm.cloud</groupId>
-        <version>2.0.15</version>
+        <version>2.0.16</version>
```

Comment:
This change should be removed. Publication automation bumps version automatically.